src/core/Akka.Persistence/Eventsourced.cs

```diff
@@ -16,44 +16,207 @@
 
 namespace Akka.Persistence
 {
-    public interface IPendingHandlerInvocation
+    internal interface IPendingHandlerInvocation
```

Comment:
Moving all of the `IPendingHandlerInvocation`s from `public` to `internal` - this is going to be part of the v1.6 API changes.